src/Sylius/Bundle/ApiBundle/spec/Serializer/OrderItemNormalizerSpec.php

```diff
@@ -66,28 +65,32 @@ function it_does_not_support_if_the_normalizer_has_been_already_called(OrderItem
         ;
     }
 
-    function it_serializes_order_item_if_item_operation_name_is_different_that_admin_get(
-        ChannelContextInterface $channelContext,
+    function it_does_not_serializes_original_price_of_an_order_item_if_section_is_admin(
         NormalizerInterface $normalizer,
         OrderItemInterface $orderItem,
-        ChannelInterface $channel,
-        ProductVariantInterface $variant,
-        ChannelPricingInterface $channelPricing
+        SectionProviderInterface $sectionProvider,
+        AdminApiSection $adminApiSection
     ): void {
         $this->setNormalizer($normalizer);
 
         $normalizer->normalize($orderItem, null, ['sylius_order_item_normalizer_already_called' => true])->willReturn([]);
+        $sectionProvider->getSection()->willReturn($adminApiSection);
 
-        $channelContext->getChannel()->willReturn($channel);
-
-        $orderItem->getVariant()->willReturn($variant);
+        $this->normalize($orderItem)->shouldNotHaveKey('originalPrice');
+    }
 
-        $variant->getAppliedPromotionsForChannel($channel)->willReturn([]);
-        $variant->getChannelPricingForChannel($channel)->willReturn($channelPricing);
+    function it_serializes_original_price_of_an_order_item_if_section_is_shop(
+        NormalizerInterface $normalizer,
+        OrderItemInterface $orderItem,
+        SectionProviderInterface $sectionProvider,
+        ShopApiSection $shopApiSection
+    ): void {
+        $this->setNormalizer($normalizer);
 
-        $channelPricing->getOriginalPrice()->willReturn(20);
+        $normalizer->normalize($orderItem, null, ['sylius_order_item_normalizer_already_called' => true])->willReturn([]);
+        $sectionProvider->getSection()->willReturn($shopApiSection);
 
-        $this->normalize($orderItem, null, [])->shouldReturn(['originalPrice' => 20]);
+        $this->normalize($orderItem)->shouldHaveKey('originalPrice');
```

Comment:
Probably should be `->shouldReturn(['originalPrice' => null])`

Reply:
`OrderItemNormalizerSpec` has been removed